wolfCLU/clu_src/x509/clu_cert_setup.c

```diff
@@ -128,7 +139,7 @@ int wolfCLU_certSetup(int argc, char** argv)
          * then write outfile */
         outfile_flag = 1;
         outfile = argv[ret+1];
-    } else {
+    } else if (text_flag == 0) {
```

Comment:
The idea here was if the text flag is set then instead of writing to a output file the output is printed to the terminal when using -x509 -inform. Something like wolfssl -rsa -sign -in <file> -inkey <file> -text does cause an error. Good catch! Fixing this now.